calcom/cal.com
# fix: allow dots in username

## What does this PR do?



Fixes https://github.com/calcom/cal.com/issues/11581

https://github.com/calcom/website/pull/624

How to test ?

- Go to profile settings and try changing the username to 'john.doe'.
OR
- create an org 
- invite a user with email (john.doe@acme.com) and then open the invite link on another browser to verify it's username is 'john.doe' and not 'john-doe' (like we did earlier)


<img width="1512" alt="Screenshot 2023-10-05 at 4 49 18 PM" src="https://github.com/calcom/cal.com/assets/53316345/46ab9b17-8130-42d5-b731-e2361efc2e06">


## Type of change
